examples/taskruns/task-multiple-output-image.yaml

```diff
@@ -73,8 +73,8 @@ spec:
     - -ce
     - |
       set -ex
-      mkdir -p /builder/home/image-outputs/builtImage1
-      cat <<EOF > /builder/home/image-outputs/builtImage1/index.json
+      mkdir -p /tekton/home/image-outputs/builtImage1
```

Comment:
NIT: I think a better change here would be to not use a fixed path at all, instead use a variable. Something for another patch probably.